### ISSUE TITLE
Obstructive Tooltip - dismiss with click

### ISSUE THREAD

**Issue by MrOggy85** [steelbrain/linter-ui-default#509]
I find this package super helpful and especially the tooltip to get a quick overview of problems with the code. 🙇 But, the Tooltip also gets in the way... A couple of examples:
1. I hover over a text and double-click it to select it. But, the tooltip have appeared and will now capture my clicks. This is only true if the cursor is in the lower bottom of the text. 
2. I hover over text as I am reading it, but I have currently several errors making the tooltip popup anywhere I put the cursor. 

TL;DR;
I would be helpful to dismiss the Tooltip on click. If you are not clicking on the button in the tooltip I would argue that the intent is to dismiss the Tooltip. Would you agree?

I am using:
Atom 1.29.0
linter-ui-default 1.7.1

**Comment by PlaidPhantom** [comment]
Seconded on this being a bit of an issue. I just ran into this issue where the tooltip appeared right under my mouse (at least, seemed to?), making it difficult to interact with the git merge plugin. maybe if the tooltip appeared further offset from the mouse location a bit? 

![image](https://user-images.githubusercontent.com/678150/117042195-46267580-acd1-11eb-8ede-5c6144d9b70b.png)
(I had to photoshop my mouse back in, but this is pretty accurate to my experience.)

I admit it's possible that the tooltip is appearing as I approach the point over the text. maybe the tooltip could be more proactive about getting out of the way of the mouse? or its background could be click-transparent? I understand this is a tricky one.
